app/validation/validator.py

```diff
@@ -1082,3 +1115,18 @@ def _get_answers_with_parent_ids(json_to_validate):
                             }
 
         return answers
+
+    @staticmethod
+    def _get_all_questions_for_block(block):
+        """ Get all questions on a block including variants"""
+        questions = []
+
+        for variant in block.get('question_variants', []):
+            if 'question' in variant:
+                questions.append(variant.get('question', {}))
```

Comment:
Me being picky but at this point the default would never be hit anyway?.
So `questions.append(variant['question'])`?

Reply:
Done